lib/core/public/setup.js

```diff
@@ -0,0 +1,12 @@
+import { getFlattenedTree, getSelectorData } from '../utils';
+
+/**
+ * Setup axe-core so axe.common functions can work properly.
+ * @param {Node} [node=document.documentElement] optional node. NOTE: passing in anything other than body or the documentElement may result in incomplete results.
+ */
+function setup(node) {
```

Comment:
I think to avoid memory leaks, we should put in an assertion here that makes sure _tree and _selectorData aren't already defined. Have it error if you call setup multiple times in a row without calling teardown.